### DISCUSSION TITLE
Clicking icons lead to different icons. Why remove the one click download anyways?

### DISCUSSION THREAD

**Discussion by EmreNurbeyler** [elrumo/macOS_Big_Sur_icons_replacements#1516] [Help with the website]
The changes to this site are bothering me, I used to be able to click an icon to download it. Now not only does it lead me to a useless App Store like page where I have to click download again, it also sometimes lead me to a landing page for a different icon.

**Comment by exp-lab-usa** [comment]
I’ve experienced this as well. The pop-up doesn’t have the save function either. Much preferred the one-click download behavior from before.
